pages/18f-agile-based-project-approach.md

```diff
@@ -18,4 +18,6 @@ Not all of our projects are greenfield; sometimes we work on legacy systems. Inc
 
 Having well-researched hypotheses beforehand allows us to be deliberate about what we build and why. Research continues throughout the agile process that we can test our hypotheses and pivot when needed. Since our work is centered around user research and user feedback, it’s important to develop participant recruiting strategies, and a research and feedback cadence that will work within the constraints of your agile process, and to ensure that the team is staffed appropriately.
 
+A common pitfall is treating agile as a silver bullet to all that ails software development; agile is viewed as risk free.  Thus when an agile projects fail, agile is blamed for it.  Agile does not eliminate risk comepletely, it provides techniques to manage risk more effectively than traditional waterfall processes.  Unlike waterfall, agile accepts that the unknowns will lead to change.  Agile treats change as an integral part of the process, as opposed to exceptions that need to be resolved via change control mechanisms.  This enables agile teams to manage risk by allowing change to drive course corrections.  However, some agile adoptions focus on the ceremonies as opposed to the intent. Incomplete, incorrect, or uninformed adoption of agile techniques can lead to an ineffective process we call “agilefall.” 18F has written a blog post about [how to avoid this pitfall](https://18f.gsa.gov/2015/12/29/is-your-project-using-agilefall/). Effective agile adoption will enable an organization to be nimble and respond effectively to the inevitable change that arises during software development.
```

Comment:
The first two sentences confused me a little bit. Suggest rewriting to be something like: "A common pitfall is expecting agile to be a silver bullet to all that ails software development, and to expect agile to eliminate all project risks. We've seen teams try agile and, when the practices fail to eliminate all the challenges, write off agile entirely, claiming "agile doesn't work." In reality, agile does not eliminate risk completely..." [also noting a typo in the word "completely"]

Love the rest of the paragraph.

Reply:
👍

Reply:
made said changes and merged into `agilefall`